Gemfile

```diff
@@ -1,2 +1,3 @@
 source 'https://rubygems.org'
-gemspec
\ No newline at end of file
+gemspec
+gem "concurrent-ruby", "0.9.1"
```

Comment:
@ph will help if I add a group there only for testing? I want the user who checkout this does not encounter this issue.